sdk/eventhub/azure-eventhub/azure/eventhub/_retry.py

```diff
@@ -0,0 +1,13 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from typing import Optional, Dict, Any
+
+from enum import Enum
+
+# INCLUDE?: _LOGGER = logging.getLogger(__name__)
```

Comment:
I don't think the logger is needed here, right? 
azure-core only has it b/c they also have the RetryPolicy methods in this file? https://github.com/Azure/azure-sdk-for-python/blob/6bcbe7f6897ebe140cfa5531dbc89a08d1a2f140/sdk/core/azure-core/azure/core/pipeline/policies/_retry.py#L50